bpf/lib/fileids.h

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+// Copyright Authors of Tetragon
+//
+// source code file to id mapping for metrics. See bpf_errmetrics.h
+// inspired by cilium's bpf metrics. See: https://github.com/cilium/cilium/pull/30972
+// NB: keep this file simple: only single-line comments and fileid__ entries
+
+fileid__("cgroup/bpf_cgtracker.c", 112)
```

Comment:
It's just random. I thought we might want to have `1` for bpf_execve.c or similar but maybe that doesn't make sense. I can change it.

Reply:
Yep maybe pre-generating them would also simplify the generation of the JSON on the `tetra` side? Just checking all files under BPF. So that we can just run a simple check validating that all files are here and in the `.JSON` asking the user to add it manually or automatically with a tool if he created a new file (like we do for the SPDX-License-Identifier header).